### ISSUE TITLE
WTB Orbital Suvey Rework

### ISSUE THREAD

**Issue by UsernamesAreAlwaysTaken** [SirMortimer/Kerbalism#75]
This is going to be a bitch. But imo, the survey scanner needs re-doing. It currently works as a click button= all info, regardless of where you are in the orbit.

Imo, if possible at all, it should work in a similar fashion to scansat. Scansat currently only generates planetary imaging, but does not handle resources.
Proposed re-work: Configure module, select a resource that you'd like to scan for, and generate overlays in a similar fashion to scansat (have to orbit a bunch of times, and gradually generate the overlay in similar fashion). This could bridge the gap between t2-t5 nodes, by running multiple scanners that scan for resources and generate some science.

I have no idea wether what i'm asking is even possible, but it's an idea.

**Comment by SirMortimer** [comment]
From `GameData/SCANsat/MM_Parts/SCANsat_Resource_Scanner.cfg`:

```
@PART[OrbitalScanner]:FOR[SCANsat]
{
        MODULE
        {
                name = ModuleSCANresourceScanner
                sensorType = 256
                fov = 3
                min_alt = 10000
                max_alt = 500000
                best_alt = 150000
                scanName = Ore Scan
                RESOURCE
                {
                        name = ElectricCharge
                        rate = 0.4
                }
        }

        MODULE
        {
                name = SCANresourceDisplay
                sensorType = 256
                ResourceName = Ore
        }
}
```

There are more. Looks like you can scan for pretty much any resource with scansat, all you have to do is define a scanner.

**Comment by SirMortimer** [comment]
What do you want to scan for?

```
		Nothing = 0, 		    // no data (MapTraq)
		AltimetryLoRes = 1 << 0,  // low resolution altimetry (limited zoom)
		AltimetryHiRes = 1 << 1,  // high resolution altimetry (unlimited zoom)
		Altimetry = (1 << 2) - 1, 	        // both (setting) or either (testing) altimetry
		SCANsat_1 = 1 << 2,		// Unused, reserved for future SCANsat scanner
		Biome = 1 << 3,		    // biome data
		Anomaly = 1 << 4,		    // anomalies (position of anomaly)
		AnomalyDetail = 1 << 5,	// anomaly detail (name of anomaly, etc.)
		Kethane = 1 << 6,         // Kethane
		MetallicOre = 1 << 7,             // CRP Ore
		Ore = 1 << 8,				//Stock Ore
		SolarWind = 1 << 9,       // SolarWind - He-3 - KSPI
		Uraninite = 1 << 10,        // Uranium - CRP
		Monazite = 1 << 11,        // Monazite - Thorium - KSPI
		Alumina = 1 << 12,        // Alumina - CRP - KSPI
		Water = 1 << 13,          // Water - CRP
		Aquifer = 1 << 14,        // Aquifer - CRP
		Minerals = 1 << 15,       // Minerals - CRP
		Substrate = 1 << 16,      // Substrate - CRP
		MetalOre = 1 << 17,          // Metal Ore - EPL
		Karbonite = 1 << 18,    // Karbonite - CRP
		FuzzyResources = 1 << 19,         // Low Detail Resource
		Hydrates = 1 << 20,		// Hydrates - CRP
		Gypsum = 1 << 21,		// Gypsum - CRP
		RareMetals = 1 << 22, // Exotic Minerals - CRP
		ExoticMinerals = 1 << 23,			// Dirt - CRP
		Dirt = 1 << 24,	// Rare Metals - CRP
		Borate = 1 << 25,		// Borate - KSPI
		GeoEnergy = 1 << 26,	// Geo Energy - Pathfinder
		SaltWater = 1 << 27,	// Salt Water - KSPI
		Silicates = 1 << 28,	// Silicates - KSPI
```

**Comment by SirMortimer** [comment]
BTW, the stock-like survey scanner behavior can be disabled in the SCANsat UI.

**Comment by SirMortimer** [comment]
Closed in favour of #76
